metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
+    article = models.ForeignKey(Article)
+    topic = models.ForeignKey(Topic)
```

Comment:
Why would the comment be specific to the topic, if the text being highlighted is the same across both topics? A given sentence can't be "bias" under one topic and "opinion" under another topic.